src/api.ts

```diff
@@ -34,17 +55,20 @@ export const KeyringAccountStruct = object({
    */
   methods: array(
     enums([
-      'personal_sign',
-      'eth_sign',
-      'eth_signTransaction',
-      'eth_signTypedData',
+      `${EthMethod.PersonalSign}`,
```

Comment:
i think we can just do `...Object.values(EthMethod)`, so if we choose to add any more eth methods we don't need to manually add it to the array.